website/package.json

```diff
@@ -12,10 +13,11 @@
     "@hashicorp/react-global-styles": "4.4.0",
     "@hashicorp/react-hashi-stack-menu": "^1.0.8",
     "@hashicorp/react-head": "1.1.1",
-    "@hashicorp/react-product-downloader": "4.0.2",
+    "@hashicorp/react-product-downloader": "^7.0.0",
     "@hashicorp/react-search": "^2.1.0",
     "@hashicorp/react-section-header": "2.0.0",
     "@hashicorp/react-subnav": "3.2.3",
+    "@hashicorp/react-tabs": "^2.0.0",
```

Comment:
This too - if its a dependency of `product-downloader`, it should be automatically brought in by that component

Reply:
Was having trouble here -- it is a dependency of product-downloader, but is unstyled unless the host app (Packer, in this case) imports the styles in style.css. Curious if you had any thoughts about how to fix this.